package.json

```diff
@@ -5,12 +5,15 @@
   "scripts": {
     "lint": "npm run eslint && npm run prettier",
     "lint_check": "npm run eslint_check && npm run prettier_check",
+    "lint_check_quick": "npm run eslint_check && npm run prettier_check_quick",
     "eslint": "npm run eslint_vanilla -- --fix",
     "eslint_check": "npm run eslint_vanilla",
     "eslint_vanilla": "eslint 'core/**/*.js'",
     "prettier": "npm run prettier_vanilla -- --write",
     "prettier_check": "npm run prettier_vanilla -- --check",
-    "prettier_vanilla": "prettier --bracket-spacing 'core/**/*.js' 'common/**/*.js' '*.js' 'voter-dapp/src/**/*.js' 'sponsor-dapp-v2/src/**/*.js' 'core/contracts/**/*.sol' 'financial-templates-lib/*.js' 'liquidator/**/*.js' 'disputer/**/*.js'"
+    "prettier_check_quick": "npm run pretty_quick -- --check",
+    "prettier_vanilla": "prettier --bracket-spacing 'core/**/*.js' 'common/**/*.js' '*.js' 'voter-dapp/src/**/*.js' 'sponsor-dapp-v2/src/**/*.js' 'core/contracts/**/*.sol' 'financial-templates-lib/*.js' 'liquidator/**/*.js' 'disputer/**/*.js'",
+    "pretty_quick": "pretty-quick --bracket-spacing 'core/**/*.js' 'common/**/*.js' '*.js' 'voter-dapp/src/**/*.js' 'sponsor-dapp-v2/src/**/*.js' 'core/contracts/**/*.sol' 'financial-templates-lib/*.js' 'liquidator/**/*.js' 'disputer/**/*.js'"
```

Comment:
This list is already kind of a nightmare to maintain. Do you think we could do something like this? Not sure if this actually works, but it might mean we don't have to maintain this list in two places.
```
"pretty_quick": "COMMAND=pretty-quick npm run prettier_vanilla",
"prettier_vanilla": "$COMMAND --bracket-spacing 'core/**/*.js' 'common/**/*.js' '*.js' 'voter-dapp/src/**/*.js' 'sponsor-dapp-v2/src/**/*.js' 'core/contracts/**/*.sol' 'financial-templates-lib/*.js' 'liquidator/**/*.js' 'disputer/**/*.js'"
```

Reply:
This is a better solution. I will give this ago. First prize for me is to put the files that it checks into the `.prettierrc` file.

Reply:
Is that possible?!?!